e2e/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "sozu-e2e"
+version = "0.1.0"
+edition = "2021"
+
+[[bin]]
+name = "sozu-e2e"
+
+[dependencies]
+sozu-lib = { path = "../lib" }
+sozu-command-lib = { path = "../command" }
+slab = "0.4.7"
```

Comment:
Could you add the `^` selector, it does not change the behaviour, but still add more semantic.